src/TLuaInterpreter.cpp

```diff
@@ -13070,6 +13070,75 @@ int TLuaInterpreter::unzipAsync(lua_State *L)
     return 1;
 }
 
+// No documentation available in wiki - internal function
+void TLuaInterpreter::fillPackageInfo(const QString& packageName, bool isModule, lua_State* packageLua)
```

Comment:
>  change it in Lua - expecting it to take effect.

If someone changes it in Lua it will take effect see #4938.
The functions setPackageInfo/setModuleInfo will also allow to add custom infos for packages/modules which are pure xml and therefore don't come with a config.lua.
For custom infos (for example Homepage or License) check the example package at #4938 

I'm obviously open for better ideas as I'm also not 100% happy with this.